This was an unified sources build error.

Build bots were failing in the [259388@main](https://commits.webkit.org/259388@main) - [259392@main](https://commits.webkit.org/259392@main) interval. Likely due to [259392@main](https://commits.webkit.org/259392@main):

https://build.webkit.org/#/builders/68/builds/20517

```
FAILED: Source/WebCore/CMakeFiles/WebCore.dir/__/__/WebCore/DerivedSources/unified-sources/UnifiedSource-cbdfe323-43.cpp.o 
/bin/ccache /usr/lib/ccache/c++ -DBUILDING_GTK__=1 -DBUILDING_WEBKIT=1 -DBUILDING_WITH_CMAKE=1 -DBUILDING_WebCore -DBWRAP_EXECUTABLE=\"/bin/bwrap\" -DDBUS_PROXY_EXECUTABLE=\"/bin/xdg-dbus-proxy\" -DGETTEXT_PACKAGE=\"WebKitGTK-4.0\" -DHAVE_CONFIG_H=1 -DJSC_GLIB_API_ENABLED -DPAS_BMALLOC=1 -DSTATICALLY_LINKED_WITH_PAL -DUSE_SYSTEM_EGL -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/WebCore/DerivedSources -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/WebGPU -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/airplay -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/applepay -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/applepay/paymentrequest -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/applicationmanifest -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/async-clipboard -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/audiosession -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/badge -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/beacon -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/cache -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/compression -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/contact-picker -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/cookie-consent -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/credentialmanagement -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/encryptedmedia -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/encryptedmedia/legacy -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/entriesapi -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/fetch -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/filesystemaccess -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/geolocation -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/highlight -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/indexeddb -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/indexeddb/client -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/indexeddb/server -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/indexeddb/shared -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/mediacapabilities -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/mediacontrols -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/mediarecorder -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/mediasession -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/mediasource -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/mediastream -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/model-element -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/model-element/dummy -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/navigatorcontentutils -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/notifications -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/paymentrequest -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/permissions -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/pictureinpicture -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/plugins -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/push-api -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/remoteplayback -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/reporting -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/screen-wake-lock -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/speech -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/storage -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/streams -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/web-locks -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webaudio -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webauthn -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webauthn/cbor -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webauthn/fido -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webcodecs -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webdatabase -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webdriver -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/websockets -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webxr -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/accessibility -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/accessibility/isolatedtree -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/animation -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/bindings -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/bindings/js -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/bridge -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/bridge/c -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/bridge/jsc -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/contentextensions -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/crypto -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/crypto/algorithms -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/crypto/keys -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/crypto/parameters -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css/calc -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css/color -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css/parser -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css/query -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css/typedom -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css/typedom/color -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css/typedom/numeric -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/css/typedom/transform -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/cssjit -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/display -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/display/compositing -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/display/css -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/dom -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/dom/messageports -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/domjit -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/editing -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/fileapi -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/history -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/html -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/html/canvas -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/html/forms -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/html/parser -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/html/shadow -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/html/track -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/inspector -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/inspector/agents -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/inspector/agents/page -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/inspector/agents/worker -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts/block -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts/block/tablewrapper -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts/flex -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/floats -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts/inline -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts/inline/display -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts/inline/invalidation -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts/inline/text -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/integration -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/integration/flex -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/integration/inline -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/layouttree -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/layout/formattingContexts/table -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/loader -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/loader/appcache -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/loader/archive -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/loader/archive/mhtml -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/loader/cache -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/loader/icon -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/mathml -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/page -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/page/csp -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/page/scrolling -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/animation -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/audio -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/calc -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/encryptedmedia -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/gamepad -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/cpu/arm -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/cpu/arm/filters -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/displaylists -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/filters -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/filters/software -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/controls -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/iso -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/opentype -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/transforms -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/mediacapabilities -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/mediarecorder -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/mediastream -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/mock -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/mock/mediasource -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/network -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/sql -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/text -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/xr -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/plugins -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/rendering -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/rendering/line -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/rendering/mathml -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/rendering/shapes -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/rendering/style -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/rendering/svg -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/rendering/updating -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/replay -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/storage -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/style -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/svg -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/svg/animation -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/svg/graphics -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/svg/graphics/filters -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/svg/properties -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/websockets -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/workers -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/workers/service -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/workers/service/context -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/workers/service/server -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/workers/shared -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/workers/shared/context -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/worklets -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/xml -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/xml/parser -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/gamepad -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/angle -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/lcms -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/ThirdParty/xdgmime/src -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/cairo -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/freetype -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/harfbuzz -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/harfbuzz/ng -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/mediastream/gstreamer -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/gstreamer -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/gstreamer/mse -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/gstreamer/eme -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/gstreamer -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/mediarecorder/gstreamer -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/audio/gstreamer -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/avif -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/bmp -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/gif -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/ico -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/jpeg -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/jpeg2000 -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/jpegxl -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/png -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/image-decoders/webp -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/network/soup -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/texmap -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/nicosia -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/page/scrolling/nicosia -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/texmap/coordinated -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/nicosia/cairo -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/nicosia/texmap -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/accessibility/atspi -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/crypto/openssl -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/page/gtk -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/adwaita -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/audio/glib -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/generic -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/glib -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/gtk -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/egl -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/epoxy -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/glx -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/gbm -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/gtk -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/libwpe -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/opengl -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/wayland -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/graphics/x11 -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/mediastream/gtk -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/mediastream/gstreamer -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/network/glib -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/platform/text/gtk -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/ANGLE/Headers -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/JavaScriptCore/Headers -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/JavaScriptCore/PrivateHeaders -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/JavaScriptCoreGLib/Headers -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/JavaScriptCoreGLib/DerivedSources -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/WTF/Headers -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/bmalloc/Headers -I/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/PAL/Headers -isystem /usr/include/gstreamer-1.0 -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/orc-0.4 -isystem /usr/lib/x86_64-linux-gnu/gstreamer-1.0/include -isystem /usr/include/gudev-1.0 -isystem /usr/include/libdrm -isystem /usr/include/libsoup-2.4 -isystem /usr/include/enchant-2 -isystem /usr/include/gio-unix-2.0 -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/libsecret-1 -isystem /usr/include/libxml2 -isystem /usr/include/cairo -isystem /usr/include/freetype2 -isystem /usr/include/harfbuzz -isystem /usr/include/openjpeg-2.4 -isystem /usr/include/avif -isystem /usr/include/gtk-3.0 -isystem /usr/include/at-spi2-atk/2.0 -isystem /usr/include/at-spi-2.0 -isystem /usr/include/dbus-1.0 -isystem /usr/lib/x86_64-linux-gnu/dbus-1.0/include -isystem /usr/include/pango-1.0 -isystem /usr/include/fribidi -isystem /usr/include/atk-1.0 -isystem /usr/include/pixman-1 -isystem /usr/include/uuid -isystem /usr/include/gdk-pixbuf-2.0 -isystem /usr/include/libpng16 -isystem /usr/include/wpe-1.0 -isystem /usr/include/libmanette -fdiagnostics-color=always -Wextra -Wall -pipe -fmax-errors=20 -Wno-odr -Wno-stringop-overread -Wno-stringop-overflow -Wno-nonnull -Wno-array-bounds -Wno-expansion-to-defined -Wno-noexcept-type -Wno-uninitialized -Wno-mismatched-new-delete -Wno-psabi -Wno-misleading-indentation -Wno-maybe-uninitialized -Wundef -Wpointer-arith -Wmissing-format-attribute -Wformat-security -Wcast-align -Wno-tautological-compare  -fno-strict-aliasing -fno-exceptions -fno-rtti -O3 -DNDEBUG -fPIC -fvisibility=hidden -DAVIF_DLL -pthread -DWPE_ENABLE_XKB=1 -std=c++20 -MD -MT Source/WebCore/CMakeFiles/WebCore.dir/__/__/WebCore/DerivedSources/unified-sources/UnifiedSource-cbdfe323-43.cpp.o -MF Source/WebCore/CMakeFiles/WebCore.dir/__/__/WebCore/DerivedSources/unified-sources/UnifiedSource-cbdfe323-43.cpp.o.d -o Source/WebCore/CMakeFiles/WebCore.dir/__/__/WebCore/DerivedSources/unified-sources/UnifiedSource-cbdfe323-43.cpp.o -c /home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/WebCore/DerivedSources/unified-sources/UnifiedSource-cbdfe323-43.cpp
In file included from /home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/WebKitBuild/Release/WebCore/DerivedSources/unified-sources/UnifiedSource-cbdfe323-43.cpp:1:
/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webaudio/OfflineAudioDestinationNode.cpp: In member function ‘WebCore::OfflineAudioDestinationNode::RenderResult WebCore::OfflineAudioDestinationNode::renderOnAudioThread()’:
/home/buildbot/worker/GTK-Linux-64-bit-Release-Ubuntu-LTS-Build/build/Source/WebCore/Modules/webaudio/OfflineAudioDestinationNode.cpp:187:75: error: invalid use of incomplete type ‘class JSC::GenericTypedArrayView<JSC::Float32Adaptor>’
  187 |             float* destination = m_renderTarget->channelData(channelIndex)->data();
```

--- a/Source/WebCore/Modules/webaudio/OfflineAudioDestinationNode.cpp
+++ b/Source/WebCore/Modules/webaudio/OfflineAudioDestinationNode.cpp
@@ -38,6 +38,8 @@
 #include "HRTFDatabaseLoader.h"
 #include "OfflineAudioContext.h"
 #include "WorkerRunLoop.h"
+#include <JavaScriptCore/GenericTypedArrayViewInlines.h>
+#include <JavaScriptCore/JSGenericTypedArrayViewInlines.h>
 #include <algorithm>
 #include <wtf/IsoMallocInlines.h>
 #include <wtf/MainThread.h>

